### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by arafay26**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -279,21 +279,21 @@ <h2>Menu</h2>
...
             </div>
-          </div>
+          </div> -->
 
-        </div>
+        <!-- </div>
 
-      </div>
-    </section><!-- End Menu Section -->
+      </div> -->
```
Comment (SyedAzeemJaved) [inline index.html:297]: Why are there two "menu" sections?

**Review by SyedAzeemJaved — CHANGES_REQUESTED** [review]
Remove the extra "menu" section, line number 173.
Remember to remove both the opening and closing tags.

File: index.html (hunk 1 above)
Comment (arafay26) [reply index.html:297]: > Why are there two "menu" sections?
while we are in the edit mode there is only one menu section. 
line287:  </section> commented

**Review by SyedAzeemJaved — CHANGES_REQUESTED** [review]
Line number 137.

Why is "End About Section" not commented?

**Comment by SyedAzeemJaved** [comment]
Remove unnecessary section(s) which you feel will not be needed.

**Comment by uzair123-ai** [comment]
> Remove unnecessary section(s) which you feel will not be needed.

K

**Pull request merged**